omero/sysadmins/limitations.rst

```diff
@@ -64,12 +65,12 @@ files before importing into OMERO. A number of suitable tools are available such
 `aicsimageio <https://github.com/AllenCellModeling/aicsimageio>`_,
 `libvips <https://github.com/libvips/libvips>`_ and `QuPath <https://qupath.github.io/>`_.
 
-Large images with floating-point pixel data
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Large images with floating-point or double pixel data types
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 :model_doc:`Pyramids <omero-pyramid/>` of image tiles are currently not
-generated for images with floating-point pixel data, meaning the imported
-image will be scrambled if it is over the size threshold mentioned above.
+generated for images with floating-point or double pixel data types, meaning the viewing of the imported
+images will be compromised if it is over the size threshold mentioned above.
```

Comment:
Main caveat with the usage of `compromised` is that it might be understood with other implications esp. in terms of security.

Reply:
This can replace the last paragraph below